std_msgs/Header header
MaskKpts[] objects